guides/common/attributes-base.adoc

```diff
@@ -29,6 +29,24 @@
 :RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
 :RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
 
+//Added by Marie
+:SatelliteSub: Red Hat Satellite Infrastructure Subscription
+// :RepoRHEL7Server: rhel-7-server-satellite
+// :RepoRHEL8Server: ???
+//:RepoRHEL7ServerSoftwareCollections: rhel-server-rhscl-7-rpms - still valid??
```

Comment:
To answer the question: on RHEL 7 we'll continue to use SCLs AFAIK so RH SCL is still needed. On EL8 there are no SCLs so there it will be irrelevant.

Reply:
Thank you, @ekohl ! So I'll uncomment the RHSCL piece. In the meantime,  I'm also reaching out to particular stakeholders for other pieces of information that are missing. Thank you!